webcompat/static/js/lib/bugform.js

```diff
@@ -267,7 +267,7 @@ function BugForm() {
   };
 
   this.storeClickedButton = function(event) {
-    this.clickedButton = event.target.value;
```

Comment:
Also this... maybe at one point I had a `value` prop on the button, I don't remember. 🙈